court_scraper/base/_last_date.py

```diff
@@ -0,0 +1,41 @@
+from datetime import timedelta, datetime
```

Comment:
We should rename the module so to remove the underscore. Underscores aren't typically used in module names, but *are* used to denote private functions or methods. In this case, let's remove the underscore on any modules that have (and of course fix the related imports).